### PR TITLE
Create 19-6.md

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/section19/19-6.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.OutputInstance.unmet_load_hours_cooling`  
+
+        **Rule Assertion:** 
+        - Case 1: If the heating + cooling unmet load hours is 300 or less then pass: `if unmet_load_hours_heating + unmet_load_hours_cooling <= 300: outcome = "PASS"`   
+        - Case 2: Else, fail: `Else: outcome = "Fail"`  
+
+**Notes/Questions:**  
+1. Does this apply at the ruleset model instance level as I created it? I wasn't sure how to reference the output schema. I took a shot at it here. 
+2. Do we check this for each rotation of the baseline model or just 0 degrees (i.e., as-designed orientation)? When I do reviews I only check the design orientation but I want to make sure that is correct.
```
Comment (weilixu) [inline docs/section19/19-6.md:33]: It seems it shall be depends on whether the `unmet_load_hours` is one metric of `baseline building performance`?
We may need to add a check for rotation exceptions if averaging the hours.

Comment (claperle) [reply docs/section19/19-6.md:33]: Per call, MK agree with JG that okay to just check as-designed rotation.

File: docs/section19/19-6.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+**Function Calls:**  None  
+
+## Rule Logic:   
+**Applicability Check 1:**  
+- For each RulesetModelInstance in ASHRAE229.ruleset_model_instance : `for RulesetModelInstance in ASHRAE229.ruleset_model_instance:`  
+    - Check if the ruleset model instance is the no rotation baseline model instance: `If RulesetModelInstance.RulesetModelOptions2019ASHRAE901.ruleset_model_type = "BASELINE_0":`  
+        - Get the heating unmet load hours for the rule set model instance: `unmet_load_hours_heating = RulesetModelInstance.output.OutputInstance.unmet_load_hours_heating`  
+        - Get the cooling unmet load hours for the rule set model instance: `unmet_load_hours_cooling = RulesetModelInstance.output.OutputInstance.unmet_load_hours_cooling`   
+        - Get the coincident unmet load hours for the rule set model instance: `coincident_unmet_load_hours = RulesetModelInstance.output.OutputInstance.unmet_load_hours`
```
Comment (supriyagoel) [inline docs/section19/19-6.md:26]: same comments as #688, should UMLH when occupied be used for H and C and should unmet_load_hrs be clarified as hrs when zone is occupied

File: docs/section19/19-6.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        - Case 1: If the coincident unmet load hours are less than or equal to 300 and not equal to Null then pass: `if coincident_unmet_load_hours != Null and coincident_unmet_load_hours <= 300: outcome = "Pass"`  
+        - Case 2: Elif the heating + cooling unmet load hours less than or equal to 300 and neither is equal to Null then pass: `elif unmet_load_hours_heating != Null and unmet_load_hours_cooling != Null and (unmet_load_hours_heating + unmet_load_hours_cooling <= 300): outcome = "Pass"`  
+        - Case 3: Elif the coincident unmet load hours are greater than 300 and not equal to Null then fail: `elif coincident_unmet_load_hours != Null and coincident_unmet_load_hours > 300: outcome = "Fail"`  
+        - Case 4: Elif the heating + cooling unmet load hours is less than or equal to 300 and neither is equal to Null then pass: `elif unmet_load_hours_heating != Null and unmet_load_hours_cooling != Null and (unmet_load_hours_heating + unmet_load_hours_cooling <= 300): outcome = "Pass"`          
+        - Case 5: Elif the heating + cooling unmet load hours greater than 600 then fail: `elif unmet_load_hours_heating + unmet_load_hours_cooling > 600: outcome = "FAIL"`
```
Comment (supriyagoel) [inline docs/section19/19-6.md:33]: same question as #688 what is the difference between case 2 and case 4

File: docs/section19/19-6.md (hunk 2 above)
Comment (claperle) [reply docs/section19/19-6.md:26]: Let's discuss on the call. App G does not separate occupied from unoccupied unmet load hours (as far as I know) so we did not include those data elements. I was not sure why they were included in the schema.

**Review by mkarpman — APPROVED** [review]

File: docs/section19/19-6.md (hunk 2 above)
Comment (claperle) [reply docs/section19/19-6.md:26]: Per call on 1/12/2023 - as the RDS is written is good. No changes needed.

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**